src/test/java/org/candlepin/async/tasks/InactiveConsumerCleanerJobTest.java

```diff
@@ -124,4 +136,46 @@ private void setRetentionDaysConfiguration(String configurationName, int retenti
             configurationName);
         this.config.setProperty(configuration, String.valueOf(retentionDays));
     }
+
+    private void createMD5Function() {
+        // We only want to create the function once as it will error if we try to create it again.
+        if (isMd5FunctionCreated) {
+            return;
+        }
+
+        isMd5FunctionCreated = true;
+        Session session = this.getCurrentSession();
+        String sql =
+            "CREATE FUNCTION md5(VARCHAR(128)) " +
+            "RETURNS VARCHAR(256) " +
+            "LANGUAGE JAVA " +
+            "DETERMINISTIC " +
+            "NO SQL " +
+            "EXTERNAL NAME 'CLASSPATH:org.candlepin.async.tasks.InactiveConsumerCleanerJobTest.md5Encode'";
+
+        session.createSQLQuery(sql)
+            .executeUpdate();
+    }
+
+    /**
+     * Used to create an MD5 hash based on a given input. This is currently being used
+     * because the hsqldb no longer has the MD5 class.
+     *
+     * @param input - used to generate the MD5 hash.
+     * @return a generated MD5 hash.
+     * @throws RuntimeException
+     */
+    public static final String md5Encode(String input) throws RuntimeException {
+        byte[] data;
+        try {
+            data = input.getBytes("ISO-8859-1");
```

Comment:
It's a moot point if we don't actually need to hash, but we'd generally want to use UTF-8 here rather than 8859. Our actual DB text fields are all UTF8, so our logic should match/reflect that.